backend/src/index.ts

```diff
@@ -1,10 +1,11 @@
-import express, { Request, Response } from 'express';
-import mongoose, { Document, Model, Schema } from 'mongoose';
 import bcrypt from 'bcrypt';
+import cookieParser from 'cookie-parser';
 import cors from 'cors';
 import dotenv from 'dotenv';
-import cookieParser from 'cookie-parser';
+import express, { Request, Response } from 'express';
 import jwt from 'jsonwebtoken';
+import Db from './models/dbConnection';
```

Comment:
Could we keep the `databaseConnection` file, but move it to the same directory as this file?

To me, database connection logic doesn't really belong in a `models` folder. Generally, models refer to items that are stored in the database, such as books or users.

Reply:
Good question. Actually, I'm confused about whether to put it. Now I have got answers.